word-search-page/word-search.js

```diff
@@ -0,0 +1,36 @@
+//Selectors
+function get(id1) {
+    return document.getElementById(id1);
+}
+let inputValue = get("search-input");
+let searchButton = get("search-button");
+let searchResult = get("search-result");
+let addResult = document.createElement('li');
+let word = inputValue.value;
+var verse;
+searchButton.addEventListener("click", search );
+
+
+function search(){
+    let url = "https://unpkg.com/quran-json@1.0.1/json/quran/en.json";
+    fetch(url)
+
+    .then(response=>{
+        findConsole = response.json();
+        console.log(findConsole);
```

Comment:
you will not get data here because the promise state of fetching data is still pending , this refers to the async nature of JavaScript that will not block the rest of the code due to async-actions , so here `console.log` will be executed before data fetching.